lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -114,6 +114,15 @@ class MongoClientInterface {
             !Number.isNaN(process.env.CONCURRENT_CURSORS))
             ? Number.parseInt(process.env.CONCURRENT_CURSORS, 10)
             : CONCURRENT_CURSORS;
+
+        this.bucketVFormatCache = new LRUCache(constants.maxCachedBuckets);
+        this.defaultBucketKeyFormat = [BUCKET_VERSIONS.v0, BUCKET_VERSIONS.v1]
+            .includes(constants.defaultBucketVFormat) ? constants.defaultBucketVFormat
+            : BUCKET_VERSIONS.v0;
```

Comment:
should this not default to v1 ?

btw, you got 2 places where the default value is defined : here and in the "constant" defintion. it would be better to have a single default value.

Reply:
it should yes, fixed that

Reply:
For the context, in S3C we leave it to v0 for now as a default because we will introduce the feature as NFI, to make sure early adopters do not have issues before making it available to every customer.

For Artesca the context is different since there are far fewer customers and less stored data, and you want customers to benefit from v1 ASAP, so I think it is good to enable v1 by default, but just so you know.